airflow/contrib/hooks/gcp_api_base_hook.py

```diff
@@ -24,13 +24,16 @@
 import google.auth
 import google_auth_httplib2
 import google.oauth2.service_account
+import os
+import tempfile
 
 from airflow.exceptions import AirflowException
 from airflow.hooks.base_hook import BaseHook
 from airflow.utils.log.logging_mixin import LoggingMixin
 
 
 _DEFAULT_SCOPES = ('https://www.googleapis.com/auth/cloud-platform',)
+G_APP_CRED = "GOOGLE_APPLICATION_CREDENTIALS"
```

Comment:
Can you change a constant name? The current name is an abbreviation of the value, but it does not describe its purpose.

Reply:
Done. On a second thought, I am going to make this private too as it's only used in gcp_api_base_hook.